grails-app/controllers/org/pih/warehouse/product/ProductSupplierController.groovy

```diff
@@ -26,6 +29,7 @@ class ProductSupplierController {
     def documentService
     def identifierService
     ProductSupplierDataService productSupplierGormService
+    ProductSupplierService productSupplierService
```

Comment:
huh, that's interesting that we didn't have `productSupplierService` injected in the `productSupplierController` :smile: :smile: